data/localizations/categories/en.yml

```diff
@@ -31055,6 +31055,45 @@ en:
     vp-2-2-1-2-3:
       name: Vans
       context: Vehicles & Parts > Vehicles > Motor Vehicles > Cars, Trucks & Vans > Non-Electric Vehicles > Vans
+    vp-2-2-1-3:
+      name: Hybrid Vehicles
+      context: Vehicles & Parts > Vehicles > Motor Vehicles > Cars, Trucks & Vans > Hybrid Vehicles
+    vp-2-2-1-3-1:
+      name: Hybrid Cars
+      context: Vehicles & Parts > Vehicles > Motor Vehicles > Cars, Trucks & Vans > Hybrid Vehicles > Hybrid Cars
+    vp-2-2-1-3-2:
+      name: Hybrid Trucks
+      context: Vehicles & Parts > Vehicles > Motor Vehicles > Cars, Trucks & Vans > Hybrid Vehicles > Hybrid Trucks
+    vp-2-2-1-3-3:
+      name: Hybrid Vans
+      context: Vehicles & Parts > Vehicles > Motor Vehicles > Cars, Trucks & Vans > Hybrid Vehicles > Hybrid Vans
+    vp-2-2-1-4:
+      name: Antique & Classic Vehicles
+      context: Vehicles & Parts > Vehicles > Motor Vehicles > Cars, Trucks & Vans > Antique & Classic Vehicles
+    vp-2-2-1-4-1:
+      name: Antique & Classic Cars
+      context: Vehicles & Parts > Vehicles > Motor Vehicles > Cars, Trucks & Vans > Antique & Classic Vehicles > Antique & Classic Cars
+    vp-2-2-1-4-1-1:
+      name: Pre-War Cars
+      context: Vehicles & Parts > Vehicles > Motor Vehicles > Cars, Trucks & Vans > Antique & Classic Vehicles > Antique & Classic Cars > Pre-War Cars
+    vp-2-2-1-4-1-2:
+      name: Post-War Cars
+      context: Vehicles & Parts > Vehicles > Motor Vehicles > Cars, Trucks & Vans > Antique & Classic Vehicles > Antique & Classic Cars > Post-War Cars
+    vp-2-2-1-4-1-3:
+      name: Muscle Cars
+      context: Vehicles & Parts > Vehicles > Motor Vehicles > Cars, Trucks & Vans > Antique & Classic Vehicles > Antique & Classic Cars > Muscle Cars
+    vp-2-2-1-4-1-4:
+      name: Hot Rod Cars
+      context: Vehicles & Parts > Vehicles > Motor Vehicles > Cars, Trucks & Vans > Antique & Classic Vehicles > Antique & Classic Cars > Hot Rod Cars
+    vp-2-2-1-4-1-5:
+      name: Vintage Cars
+      context: Vehicles & Parts > Vehicles > Motor Vehicles > Cars, Trucks & Vans > Antique & Classic Vehicles > Antique & Classic Cars > Vintage Cars
```

Comment:
As discussed, this brings some likely duplication in products like a modified 1923 Ford T-Bucket that could fit in three categories: Pre-War, Hot Rod, Vintage.

Aligned on one single category called `Vintage & Classic Cars` (as "antique" and "classic" are often used interchangeably, but "Vintage" is often defined as _from the period of 1919 to 1930_ versus "Classic" as _25 years or older_). Also an attribute called `car era` and a design attribute for concepts like t-bucket vs. muscle.

![image](https://github.com/user-attachments/assets/0a9efb58-4462-4396-8887-b2d1a4f190f8)